websites/I/IFTTT/presence.ts

```diff
@@ -1,277 +1,361 @@
-              presenceData.state = document.querySelector("h1").textContent;
-            } else {
-              presenceData.smallImageKey = "reading";
-              presenceData.smallImageText = (await strings).browsing;
-            }
-            break;
-          //Library
-          case "blog":
-            presenceData.state = "Library";
+        presence.setActivity();
+        return;
+    }
+  }
 
-            if (document.getElementsByClassName("story-title").length > 0) {
-              presenceData.smallImageKey = "reading";
-              presenceData.smallImageText = (await strings).reading;
+  presence.setActivity(presenceData);
+});
 
-              presenceData.details = "Library";
-              presenceData.state =
-                document.getElementsByClassName("story-title")[0].textContent;
-            }
-            break;
-          //Case studies
-          case "case_studies":
-            presenceData.state = "Case studies";
+/**
+ * Get Language Strings
+ * @returns Language Strings
+ */
+async function getStrings() {
+  return presence.getStrings(
+    {
+      search: "general.searching",
+      browsing: "general.browsing",
+      reading: "general.reading"
+    },
+    (await presence.getSetting("lang").catch(() => "en")) as string
+  );
+}
 
-            if (document.getElementsByClassName("story-title").length > 0) {
-              presenceData.smallImageKey = "reading";
-              presenceData.smallImageText = (await strings).reading;
+/**
+ * Handle status page
+ */
+async function handleStatusPage(): Promise<void> {
+  let incidents: HTMLDivElement[];
 
-              presenceData.details = "Case studies";
-              presenceData.state =
-                document.getElementsByClassName("story-title")[0].textContent;
-            }
-            break;
-          //Testimonials
-          case "testimonials":
-            presenceData.state = "Testimonials";
-            break;
-          //Contact
-          case "contact_sales":
-            presenceData.state = "Contact";
-            break;
-          //Terms of Use
-          case "terms":
-            presenceData.state = "Terms of Use";
-            break;
-        }
-      }
-      break;
+  try {
+    incidents = [
+      ...(document.querySelector<HTMLDivElement>(".unresolved-incidents")
+        .children as unknown as HTMLDivElement[])
+    ].filter(e => e.style.display !== "none");
+  } catch (e) {
+    incidents = [];
+  }
 
-    //IFTTT Help Center
-    case "help.ifttt.com":
-      presenceData.details = "Help Center";
-      if (path.length > 2) {
-        switch (path[2]) {
-          //Articles
-          case "articles":
-            presenceData.smallImageKey = "reading";
-            presenceData.smallImageText = (await strings).reading;
+  presence.setActivity({
+    details: "IFTTT Status",
+    state:
+      incidents.length > 0 ? `Unresolved incidents: ${incidents.length}` : null,
```

Comment:
Use an if instead and if it is zero it is falsy

Reply:
![grafik](https://user-images.githubusercontent.com/47316898/147494900-d85b6596-0ee7-4187-94c9-6d8ae8c39c3d.png)